ocaml/parsing/parser.mly

```diff
@@ -504,7 +504,7 @@ let lapply ~loc p1 p2 =
 let loc_map (f : 'a -> 'b) (x : 'a Location.loc) : 'b Location.loc =
   { x with txt = f x.txt }
 
-let make_ghost x = { x with loc = { x.loc with loc_ghost = true }}
+let make_ghost x = { x with loc = Location.ghostify x.loc }
```

Comment:
This could avoid allocation by testing for the ghost before proceeding.

Reply:
Addressed in antalsz#1